folding-schemes/src/folding/nova/mod.rs

```diff
@@ -857,6 +862,77 @@ pub mod tests {
         );
     }
 
+    #[test]
+    fn test_serde_nova() {
```

Comment:
should this test be under `nova/serialize.rs` since is testing the methods from that file?